pingo/examples/lcd16x2_countdown.py

```diff
@@ -0,0 +1,110 @@
+""" A countdown in a 16x2 LCD.
+It is possible to pause or reset the countdown using buttons.
+When the countdown ends buzzes continuously.
+
+Requires:
+- Adafruit_CharLCD (https://github.com/adafruit/Adafruit_Python_CharLCD)
+
+Hardware:
+- 16x2 LCD (8 digital pins)
+- Buzzer (1 digital pin)
+- 2x buttons (1 digital pin each)
+
+"""
+
+
+import time
+import pingo
+
```

Comment:
from pingo import State, Mode

should be necessary if Enum for State and Mode is accept

see https://github.com/pingo-io/pingo-py/pull/109